docs/software-platforms/lsfr/creating-vcs.md

```diff
@@ -0,0 +1,104 @@
+# Creating virtual computers with Lightsail for Research
+
+While working on an [analysis module](../../contributing-to-analyses/analysis-modules/index.md), you can develop locally or on a virtual computer with Lightsail for Research (LSfR).
+
+Virtual computers provide access to a set amount of virtual CPUs, memory, and storage through a web browser rather than a physical machine.
+Using a virtual computer allows you to develop and run analyses that require more CPUs and memory than is available on your local machine.
+
+_Note that you will need access to AWS to use LSfR._
+See [getting access to AWS](../../getting-started/accessing-resources/index.md#getting-access-to-aws) to learn how to get set up with AWS.
+
+- The virtual computers provided through LSfR use an Ubuntu operating system.
+- During setup, you will choose your desired configuration, including the amount of virtual CPUs, memory, and storage space.
+    - You also have the option to add [additional storage to your virtual computer](STUB-LINK for volumes).
+- All virtual computers will have a set of pre-installed applications needed for working with OpenScPCA, including:
+    - Git and GitKraken
+    - The AWS command line interface
+    - R
+    - Conda
+
+!!! note "More on virtual computers"
+
+    Read more about [virtual computers with Lightsail for Research](https://docs.aws.amazon.com/lightsail-for-research/latest/ug/computers.html).
+
+## How to create a virtual computer
+
+Follow the below steps to create a virtual computer to use with LSfR.
+
+1. Navigate to the [access portal URL from when you set up your user in IAM Identity center](../aws/index.md#joining-iam-identity-center).
+This link should bring you to the AWS Console and home page.
+
+    Once here, be sure that you are in the `us-east-2` region, by selecting the drop-down menu in the tool bar.
+
+    <figure markdown="span">
+        ![Select region](../../img/creating-vcs-1.png){width="400"}
+    </figure>
+
+1. You will need to use the AWS Service Catalog to create virtual computers.
```

Comment:
We should note that directly creating an instance via Lightsail for Research is not supported. Mostly raising this because we're linking to LfR docs.